src/apps/loan-list/materials/utils/status-circle-icon.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { FC, ReactNode } from "react";
+import { useText } from "../../../../core/utils/text";
+
+interface StatusCircleIconProps {
+  color: string;
+  percent: number;
+  children: ReactNode;
+}
+
+const StatusCircleIcon: FC<StatusCircleIconProps> = ({
+  color,
+  percent,
+  children
+}) => {
+  const t = useText();
+
+  return (
+    <div
+      className="list-reservation__counter"
+      aria-label={t("loanListStatusCircleIconAriaLabelText")}
```

Comment:
I have on my endless todo to fix all the strings in the apps when the string interpolation works.... I'll add a todo instead, because it'll change in a future pr anyway.